config/connectors.yml.example

```diff
@@ -20,7 +20,9 @@ thread_pool:
 
 log_level: info
 
-idle_timeout: 60
+poll_interval: 3
+termination_timeout: 60
```

Comment:
I don't see where this one is used? And the name doesn't tell me about its intention.

Reply:
It is used here:
https://github.com/elastic/connectors-ruby/blob/bf9a3252cfc63fd1e1f4e0a999f396d388f3de99/lib/app/dispatcher.rb#L19